shopelectro/views/catalog.py

```diff
@@ -38,19 +39,31 @@ class CategoryTree(catalog.CategoryTree):
     category_model = models.Category
 
 
-def prepare_tile_products(products):
-    images = Image.objects.get_main_images_by_pages(
-        models.ProductPage.objects.filter(
-            shopelectro_product__in=products
+class TemplateDataAdapter:
```

Comment:
@duker33 
I still believe that such names are very bad for any classes, because they do not make any sense for us. I still would prefer to see `ProductCards/ProductTiles(...).get()` or something like that.

Decide

Also Yegor B. has an [article](https://www.yegor256.com/2015/03/09/objects-end-with-er.html) about such names :)

Reply:
@artemiy312 , my idea is not about products cards or tiles.
The idea is to systemize bunch of `prepare` methods with classes hierarchy.
But seems i don't know how to it right now.
I'll revert code related to this arch refactoring and create a separated issue about it